gap/oper.gi

```diff
@@ -1239,6 +1239,67 @@ function(digraph, u, v)
   return DIGRAPH_PATH(OutNeighbours(digraph), u, v);
 end);
 
+# DigraphShortestPath
+
+InstallMethod(DigraphShortestPath, "for a digraph and two pos ints",
+[IsDigraph, IsPosInt, IsPosInt],
+function(digraph, u, v)
+  local current, next, parent, distance, falselist, verts, nbs, n, a, b, i, path;
+
+  verts    := DigraphVertices(digraph);
+  nbs      := OutNeighbors(digraph);
+  distance := [];
+
+  if u = v and v in nbs[v] then  # Considers the trivial path from v to v
+    return [[v, v], [Position(nbs[v], v)]];
```

Comment:
Definition of trivial in the package documentation:
```
The length of a directed walk (v1,e1,v2,e2,...,en−1,vn) is equal to n−1, the number of edges it contains. A directed walk of zero length, i.e. a sequence (v) for some vertex v, is called trivial. A trivial directed walk is considered to be both a circuit and a cycle, as is the empty directed walk ().
```
By the original documentation in this PR:
```
(or a non-trivial cycle, in the case that <A>u</A> <C>=</C> <A>v</A>)
```
a loop therefore qualifies as the shortest path, since a loop is a non-trivial cycle.

However, I therefore suggest adjusting the misleading comment in the line above.